frontend/src/io-managers/persistence.ts

```diff
@@ -42,18 +50,23 @@ export function createPersistenceManager(editor: Editor, portfolio: PortfolioSta
 		);
 
 		await storeDocumentOrder();
+		await storeDocumentIndex()
 	}
 
 	async function loadDocuments() {
 		const previouslySavedDocuments = await get<Record<string, TriggerIndexedDbWriteDocument>>("documents", graphiteStore);
 		const documentOrder = await get<string[]>("documents_tab_order", graphiteStore);
+		const currentDocumentId = await get<string>("current_document_id", graphiteStore);
 		if (!previouslySavedDocuments || !documentOrder) return;
 
 		const orderedSavedDocuments = documentOrder.flatMap((id) => (previouslySavedDocuments[id] ? [previouslySavedDocuments[id]] : []));
 
 		orderedSavedDocuments?.forEach(async (doc: TriggerIndexedDbWriteDocument) => {
 			editor.handle.openAutoSavedDocument(BigInt(doc.details.id), doc.details.name, doc.details.isSaved, doc.document);
 		});
+		if (!!currentDocumentId){
```

Comment:
The `!!` is redundant here. Please also watch your formatting with there being no space between the `)` and `{`. Your editor must be not set up to format on autosave, so you might want to get `cd frontend && npm run lint-fix` to run in your post-commit hook or format on save.